webpack.config.js

```diff
@@ -2,15 +2,17 @@
 
 const merge = require('webpack-merge')
 
-const build = /production$/.test(process.env.NODE_ENV)
+const build = /:production$/.test(process.env.NODE_ENV)
```

Comment:
@m4dz it's for resolve when you launch `yarn build` or `yarn watch` without :browser or :mobile